lib/matplotlib/font_manager.py

```diff
@@ -685,6 +686,7 @@ def __init__(self,
                  stretch= None,
                  size   = None,
                  fname  = None,  # if set, it's a hardcoded filename to use
+                 mathtext_fontset = None,
```

Comment:
I don't see this kwarg documented in the class, though its documented below in `set_mathtext_fontset`

Reply:
if you dont set the mathtext_fontset it will take the value of the global parameter when it is created (that is rcParams['mathtext.fontset']), this was one of the firsts things I implemented and when I did that I didn't know how the docs worked (I though I would have to edit the docs in HTML later) so I forgot to do this part. I will make the corrections. I will also have to do the same for the Text class I believe.